Bucketeer2/Sources/Public/BKTUser.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+public struct BKTUser {
+    public let id: String
+    public internal(set) var attributes: [String: String]
+
+    public func builder() -> Builder {
+        return .init()
+    }
+
+    public mutating func custom(attributes: [String: String]) {
+        self.attributes = attributes
+    }
+
+    public class Builder {
```

Comment:
I know the Builder pattern is very common in Android development, but it might be more common for ios developers if we do like below.

```swift
let config = Config(
  apiKey: "api-key",
  apiEndpoint: "api.example.com",
  featureTag: "ios"
)
let user = User(
  user: "user-id",
  attributes: ["app_version": "1.0.0", "country": "Japan"]
)

BKTClient.initialize(config: config, user: user)
BKTClient.shared.getStringVariation(featureID: "feature-id", defaultValue: "default-value")
```

Same for `BKTConfig`.

Reply:
Changed common initializer instead of Builder pattern https://github.com/ca-dp/bucketeer-ios-sdk/pull/86/commits/144e2a02474e5861ada9727fafb82a4683956c4c